proj/pyKmeans.cpp

```diff
@@ -84,11 +98,14 @@ class pyKmeans{
 
 };
 
+}/*end namespace pybind11*/
+
 
 PYBIND11_MODULE(_kmeans, m) 
 {
     m.doc() = "K-means on image";
 
+    using namespace pybind11;

```

Comment:
If you `using namespace pybind11;`, why bother writing `py::...`?  The inconsistency in coding style makes your code harder to understand.